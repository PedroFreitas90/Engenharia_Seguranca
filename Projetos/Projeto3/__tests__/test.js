const React = require('react');
const App = require('./../ui')
const renderer = require('react-test-renderer');
function sleep(ms) {
    return new Promise(resolve => setTimeout(resolve, ms));
}
test('should add two numbers', () => {
    const componentApp = renderer.create(
        <App operation="GetCertificate" user="351 918133837"/>,
        sleep(2000)
    );
   
    let tree = componentApp.toJSON();
    expect(tree).toMatchSnapshot();
});