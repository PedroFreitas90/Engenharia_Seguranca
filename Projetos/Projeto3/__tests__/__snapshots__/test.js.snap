// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add two numbers 1`] = `
<span
  style={
    Object {
      "flexDirection": "row",
    }
  }
  unstable__transformChildren={[Function]}
>
  GetCertificate
  : 
   
</span>
`;
